WordPress/Classes/ViewRelated/Me/All Domains/View Models/AllDomainsListItemViewModel.swift

```diff
@@ -0,0 +1,60 @@
+import Foundation
+
+struct AllDomainsListItemViewModel {
+
+    let name: String
+    let description: String?
+    let status: Status?
+    let expiryDate: String?
+
+    private enum Strings {
+        static let expired = NSLocalizedString(
+            "domain.management.card.expired.label",
+            value: "Expired",
+            comment: "The expired label of the domain card in My Domains screen."
+        )
+        static let renews = NSLocalizedString(
+            "domain.management.card.renews.label",
+            value: "Renews",
+            comment: "The renews label of the domain card in My Domains screen."
+        )
+    }
+
+    typealias Domain = DomainsService.AllDomainsListItem
+    typealias Status = Domain.Status
+    typealias StatusType = DomainsService.AllDomainsListItem.StatusType
+}
+
+extension AllDomainsListItemViewModel {
```

Comment:
[Nit] I do not see the value of this extension. It's a relatively small struct, and the extension mainly contains a custom init which is traditionally placed in the struct itself. To me, the file is hard to read at the moment. For example, the init in the extension uses the Strings enum in the main struct.

Reply:
The extension wasn't added to break down a large `struct`. As you said, the `struct` is small. The extension was added to add the convenience `init(domain:)` and maintain the compiler-generated init. [Here is a nice article explaining this practice.](https://www.swiftbysundell.com/tips/struct-convenience-initializers/)

Reply:
Thanks for the refactor @salimbraksa! The file is much more readable now 👍